templates/launch.html

```diff
@@ -172,6 +99,44 @@ <h1>Launch a survey</h1>
 
 <script>
 
+    function loadMetadata() {
+        var xhttp = new XMLHttpRequest();
+        xhttp.onreadystatechange = function() {
+            if (this.readyState == 4 && this.status == 200) {
+
+                document.getElementById("survey_metadata").innerHTML = ""
+
+                var response = JSON.parse(this.responseText);
+
+                if (response.length > 0) {
+                    for (var i = 0; i < response.length; i++) {
+                        metadataField = response[i]
+
+                        var defaultValue = metadataField['name'];
+
+                        if (metadataField['default'] != "") {
+                            defaultValue = metadataField['default']
+                        }
+
+                        var metadataFieldHtml = "<div class=\"field-container\">" +
```

Comment:
This will need to have variations based on the type, right? if we have `metadataField['format']` in the response, can you render checkboxes for `boolean` types and perhaps a UUID generator can be used to make the default for `uuid` type fields?